src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -75,4 +76,52 @@ describe('InfiniteScroll', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('length when step < array', () => {
+    const allItems = Array(1000)
+      .fill()
+      .map((_, i) => `item ${i + 1}`);
```

Comment:
Since this seems like we are repeating this logic across several tests, does it make sense for us to abstract this and call it something like `simpleItems`?

Or is the direction to keep the test completely self contained?

Reply:
Yes I agree it makes sense to get rid of the repeating logic. I added `simpleItems` at the beginning of the test suite so it can be used by all 3 of the tests instead of being duplicated